### PR TITLE
feat(Icons): Adds new icons for Form updates

### PR DESCRIPTION
**Pull request by dreamwasp**
### Overview
Adds new iIcons for Form updates

### PR Checklist

- [x] Related to designs: https://www.figma.com/file/9Ke4l9EdIup7kiB3KsbEHx/Form-Inputs-Components?node-id=1218%3A0
- [x] Related to JIRA ticket: gm-152
- [x] I have run this code to verify it works
- [x] This PR includes unit tests for the code change

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@4.1.1-alpha.d39e4f.0<br/>@codecademy/gamut-kit@0.2.1-alpha.d39e4f.0<br/>@codecademy/gamut-labs@10.4.7-alpha.d39e4f.0<br/>@codecademy/gamut@26.0.2-alpha.d39e4f.0<br/>@codecademy/styleguide@30.0.2-alpha.d39e4f.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6042810a92199c1295934b6d--gamut-preview.netlify.app>https://6042810a92199c1295934b6d--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6042810a92199c1295934b6d>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@4.1.1-alpha.86476c.0<br/>@codecademy/gamut-kit@0.2.4-alpha.86476c.0<br/>@codecademy/gamut-labs@10.5.3-alpha.86476c.0<br/>@codecademy/gamut@26.0.2-alpha.86476c.0<br/>@codecademy/styleguide@30.1.3-alpha.86476c.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6042a59e93462c00cd67f5d4--gamut-preview.netlify.app>https://6042a59e93462c00cd67f5d4--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6042a59e93462c00cd67f5d4>Deploy Logs</a></p>

File: packages/styleguide/stories/Atoms/Icons/constants.tsx [hunk 1]
```diff
@@ -23,6 +23,9 @@ export const VENDOR_ICONS = pick(
     'FaviconIcon',
     'FaviconSolidIcon',
     'JavaIcon',
+    'AmexIcon',
```
Comment (codecaaron) [inline packages/styleguide/stories/Atoms/Icons/constants.tsx:26]: Can you update the import here to `import * from '@codecademy/gamut-icons/src/regular'`?  I forgot to do it the last time :)

File: packages/gamut-icons/src/svg/mini/mini-info-outline-icon.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" version="1.1" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:svgjs="http://svgjs.com/svgjs" viewBox="0 0 24 24" width="140" height="140"><title>mini-info-outline</title><path d="M0.500 7.000 A6.500 6.500 0 1 0 13.500 7.000 A6.500 6.500 0 1 0 0.500 7.000 Z" fill="none" stroke="#000000" stroke-linecap="round" stroke-linejoin="round"></path><path d="M7 7L7 10.5" fill="none" stroke="#000000" stroke-linecap="round" stroke-linejoin="round"></path><path d="M6.500 4.500 A0.500 0.500 0 1 0 7.500 4.500 A0.500 0.500 0 1 0 6.500 4.500 Z" fill="none" stroke="#000000" stroke-linecap="round" stroke-linejoin="round"></path></svg>
```
Comment (codecaaron) [inline packages/gamut-icons/src/svg/mini/mini-info-outline-icon.svg:1]: I think something happened to this one, might be time to update the docs here, I think they are pretty stale now :(

Comment (codecaaron) [reply packages/gamut-icons/src/svg/mini/mini-info-outline-icon.svg:1]: Could you add the OG asset for it?

Comment (dreamwasp) [reply packages/gamut-icons/src/svg/mini/mini-info-outline-icon.svg:1]: oooo yeah, i'll fix it

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@4.1.1-alpha.3c7a25.0<br/>@codecademy/gamut-kit@0.2.4-alpha.3c7a25.0<br/>@codecademy/gamut-labs@10.5.3-alpha.3c7a25.0<br/>@codecademy/gamut@26.0.2-alpha.3c7a25.0<br/>@codecademy/styleguide@30.1.3-alpha.3c7a25.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6042ae87707d1c08d9a3da73--gamut-preview.netlify.app>https://6042ae87707d1c08d9a3da73--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6042ae87707d1c08d9a3da73>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@4.1.1-alpha.844b4b.0<br/>@codecademy/gamut-kit@0.2.4-alpha.844b4b.0<br/>@codecademy/gamut-labs@10.5.3-alpha.844b4b.0<br/>@codecademy/gamut@26.0.2-alpha.844b4b.0<br/>@codecademy/styleguide@30.1.3-alpha.844b4b.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6042b4d51d11bb0b0fada447--gamut-preview.netlify.app>https://6042b4d51d11bb0b0fada447--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6042b4d51d11bb0b0fada447>Deploy Logs</a></p>

File: packages/styleguide/stories/Atoms/Icons/constants.tsx (hunk 1 above)
Comment (dreamwasp) [reply packages/styleguide/stories/Atoms/Icons/constants.tsx:26]: this change gives me a lint error 🙃
image

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60479609c58b06d514574441--gamut-preview.netlify.app>https://60479609c58b06d514574441--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60479609c58b06d514574441>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@4.1.1-alpha.ca8022.0<br/>@codecademy/gamut-kit@0.2.7-alpha.ca8022.0<br/>@codecademy/gamut-labs@11.0.3-alpha.ca8022.0<br/>@codecademy/gamut@26.1.1-alpha.ca8022.0<br/>@codecademy/styleguide@30.2.2-alpha.ca8022.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6047977cd089e6007d9956db--gamut-preview.netlify.app>https://6047977cd089e6007d9956db--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6047977cd089e6007d9956db>Deploy Logs</a></p>

**Review by codecaaron — APPROVED** [review]
🔥

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@4.2.1-alpha.f089ba.0<br/>@codecademy/gamut-kit@0.2.8-alpha.f089ba.0<br/>@codecademy/gamut-labs@11.0.4-alpha.f089ba.0<br/>@codecademy/gamut@26.2.1-alpha.f089ba.0<br/>@codecademy/styleguide@30.3.1-alpha.f089ba.0<br/></pre>

**Pull request merged**